handle-form.js

```diff
@@ -0,0 +1,85 @@
+const usernameInput = document.getElementById("username");
+const emailInput = document.getElementById("email");
+const passwordInput = document.getElementById("password");
+const repasswordInput = document.getElementById("repassword");
+
+const regexChars = {
+  uppercase: /[A-Z]/,
+  lowercase: /[a-z]/,
+  digit: /[0-9]/,
+  specialChar: /[!@#$%^&*()_\-+=,.?]/,
+  email: /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/,
+  space: /\s/,
+};
+
+var validate = 0;
```

Comment:
anh nghĩ nên có 1 biến errors, lưu các lỗi của em
```suggestion
const errors = {
   username: "",
   email: "",
   ...
};
const isInvalid = Object.keys(errors).length > 0
```